include/oneapi/dpl/experimental/kt/esimd_radix_sort_utils.h

```diff
@@ -119,13 +130,17 @@ scan(sycl::ext::intel::esimd::simd<T, 16> src)
 }
 
 // get bits value (bucket) in a certain radix position
-template <::std::uint32_t __radix_mask, typename _T, int _N>
-sycl::ext::intel::esimd::simd<_T, _N>
+template <::std::uint16_t __radix_mask, typename _T, int _N>
+sycl::ext::intel::esimd::simd<::std::uint16_t, _N>
 __get_bucket(sycl::ext::intel::esimd::simd<_T, _N> __value, ::std::uint32_t __radix_offset)
 {
-    return (__value >> __radix_offset) & sycl::ext::intel::esimd::simd<_T, _N>(__radix_mask);
+    return sycl::ext::intel::esimd::simd<::std::uint16_t, _N>(__value >> __radix_offset) & __radix_mask;
 }
 
+template <typename T, bool __is_ascending>
+inline constexpr T __sort_neutral =
```

Comment:
I like the name of the value given its connection to abstract algebra. You can also consider `__sort_identity`, because `identity` seems to be used more frequently, e.g. it is used in SYCL specification.